addons/hr/views/res_config_settings_views.xml

```diff
@@ -9,34 +9,32 @@
             <xpath expr="//form" position="inside">
                 <app data-string="Employees" string="Employees" name="hr" groups="hr.group_hr_manager">
                     <block title="Employees" name="employees_setting_container">
-                        <setting id="presence_control_setting" title="Presence of employees" string="Presence Control">
+                        <setting id="presence_control_setting"
+                            help="Which information will define the presence status of your employees."
+                            title="Presence of employees" string="Presence Display">
                             <div class="content-group" name="hr_presence_options">
                                 <div class="d-flex">
-                                    <field name="module_hr_attendance" class="ml16"/>
-                                    <label for="module_hr_attendance" class="o_light_label"/>
-                                </div>
-                                <div class="d-flex">
-                                    <field name="hr_presence_control_login" class="ml16"/>
-                                    <label for="hr_presence_control_login" class="o_light_label"/>
+                                    <field name="hr_presence_control" widget="radio"/>
                                 </div>
                             </div>
                         </setting>
-                        <setting id="presence_reporting_setting" help="Presence reporting screen, email and IP address control." title="Advanced presence of employees">
-                            <field name="module_hr_presence"/>
-                            <div class="d-flex mt-1" invisible="not module_hr_presence">
-                                <field name="hr_presence_control_email" class="ml16"/>
-                                <label for="hr_presence_control_email" class="o_light_label"/>
-                            </div>
-                            <div class="d-flex ml32" invisible="not module_hr_presence or not hr_presence_control_email">
-                                <span class="flex-shrink-0 ml8 me-2">Minimum number of emails to send</span>
+                        <setting invisible="hr_presence_control != 'user_status'"
```

Comment:
It doesn't work with the attendance button on :) Maybe do this instead to make it more flexible for eventual choice additions: 
```suggestion
                        <setting invisible="hr_presence_control in ['user_email', 'user_ip']"
```